util.cpp:499:12: error: enumeration value 'ESP_MATTER_VAL_TYPE_NULLABLE_BOOLEAN' not handled in switch [-Werror=switch]

Hello,

I am facing this issue when I build:
```bash
/home/user/matter-esp32-dcmotor/main/src/system/util.cpp: In function 'std::string get_matter_attribute_value_string(esp_matter_attr_val_t)':
/home/user/matter-esp32-dcmotor/main/src/system/util.cpp:499:12: error: enumeration value 'ESP_MATTER_VAL_TYPE_NULLABLE_BOOLEAN' not handled in switch [-Werror=switch]
  499 |     switch (value.type) {
      |            ^
```

### Full log

```bash
user@user-virtual-machine:~/matter-esp32-dcmotor$ idf.py build
Executing action: all (aliases: build)
Running ninja in directory /home/user/matter-esp32-dcmotor/build
Executing "ninja all"...
[1/1] cd /home/user/matter-esp32-dcmotor/build/bootloader/esp-idf/esptool_py && /home/...c000 bootloader 0x1000 /home/user/matter-esp32-dcmotor/build/bootloader/bootloader.binBootloader binary size 0x6160 bytes. 0x4ea0 bytes (45%) free.
[2/12] Performing build step for 'chip_gn'ninja: no work to do.
[3/10] Building CXX object esp-idf/main/CMakeFiles/__idf_main.dir/src/system/util.cpp.objFAILED: esp-idf/main/CMakeFiles/__idf_main.dir/src/system/util.cpp.obj
ccache /home/user/.espressif/tools/xtensa-esp32-elf/esp-12.2.0_20230208/xtensa-esp32-elf/bin/xtensa-esp32-elf-g++ -DESP_PLATFORM -DIDF_VER=\"v5.1.1-577-g6b1f40b9bf\" -DMBEDTLS_CONFIG_FILE=\"mbedtls/esp_config.h\" -DNDEBUG -DSOC_MMU_PAGE_SIZE=CONFIG_MMU_PAGE_SIZE -D_GNU_SOURCE -D_POSIX_READER_WRITER_LOCKS -I/home/user/matter-esp32-dcmotor/build/config -I/home/user/matter-esp32-dcmotor/main/include -I/home/user/matter-esp32-dcmotor/main/include/device -I/home/user/matter-esp32-dcmotor/main/include/peripheral -I/home/user/matter-esp32-dcmotor/main/include/system -I/home/user/tools/esp-idf/components/newlib/platform_include -I/home/user/tools/esp-idf/components/freertos/FreeRTOS-Kernel/include -I/home/user/tools/esp-idf/components/freertos/FreeRTOS-Kernel/portable/xtensa/include -I/home/user/tools/esp-idf/components/freertos/esp_additions/include/freertos -I/home/user/tools/esp-idf/components/freertos/esp_additions/include -I/home/user/tools/esp-idf/components/freertos/esp_additions/arch/xtensa/include -I/home/user/tools/esp-idf/components/esp_hw_support/include -I/home/user/tools/esp-idf/components/esp_hw_support/include/soc -I/home/user/tools/esp-idf/components/esp_hw_support/include/soc/esp32 -I/home/user/tools/esp-idf/components/esp_hw_support/port/esp32/. -I/home/user/tools/esp-idf/components/esp_hw_support/port/esp32/private_include -I/home/user/tools/esp-idf/components/heap/include -I/home/user/tools/esp-idf/components/log/include -I/home/user/tools/esp-idf/components/soc/include -I/home/user/tools/esp-idf/components/soc/esp32 -I/home/user/tools/esp-idf/components/soc/esp32/include -I/home/user/tools/esp-idf/components/hal/esp32/include -I/home/user/tools/esp-idf/components/hal/include -I/home/user/tools/esp-idf/components/hal/platform_port/include -I/home/user/tools/esp-idf/components/esp_rom/include -I/home/user/tools/esp-idf/components/esp_rom/include/esp32 -I/home/user/tools/esp-idf/components/esp_rom/esp32 -I/home/user/tools/esp-idf/components/esp_common/include -I/home/user/tools/esp-idf/components/esp_system/include -I/home/user/tools/esp-idf/components/esp_system/port/soc -I/home/user/tools/esp-idf/components/esp_system/port/include/private -I/home/user/tools/esp-idf/components/xtensa/include -I/home/user/tools/esp-idf/components/xtensa/esp32/include -I/home/user/tools/esp-idf/components/lwip/include -I/home/user/tools/esp-idf/components/lwip/include/apps -I/home/user/tools/esp-idf/components/lwip/include/apps/sntp -I/home/user/tools/esp-idf/components/lwip/lwip/src/include -I/home/user/tools/esp-idf/components/lwip/port/include -I/home/user/tools/esp-idf/components/lwip/port/freertos/include -I/home/user/tools/esp-idf/components/lwip/port/esp32xx/include -I/home/user/tools/esp-idf/components/lwip/port/esp32xx/include/arch -I/home/user/tools/esp-matter/device_hal/device/include -I/home/user/tools/esp-matter/device_hal/led_driver/include -I/home/user/tools/esp-idf/components/driver/include -I/home/user/tools/esp-idf/components/driver/deprecated -I/home/user/tools/esp-idf/components/driver/analog_comparator/include -I/home/user/tools/esp-idf/components/driver/dac/include -I/home/user/tools/esp-idf/components/driver/gpio/include -I/home/user/tools/esp-idf/components/driver/gptimer/include -I/home/user/tools/esp-idf/components/driver/i2c/include -I/home/user/tools/esp-idf/components/driver/i2s/include -I/home/user/tools/esp-idf/components/driver/ledc/include -I/home/user/tools/esp-idf/components/driver/mcpwm/include -I/home/user/tools/esp-idf/components/driver/parlio/include -I/home/user/tools/esp-idf/components/driver/pcnt/include -I/home/user/tools/esp-idf/components/driver/rmt/include -I/home/user/tools/esp-idf/components/driver/sdio_slave/include -I/home/user/tools/esp-idf/components/driver/sdmmc/include -I/home/user/tools/esp-idf/components/driver/sigma_delta/include -I/home/user/tools/esp-idf/components/driver/spi/include -I/home/user/tools/esp-idf/components/driver/temperature_sensor/include -I/home/user/tools/esp-idf/components/driver/touch_sensor/include -I/home/user/tools/esp-idf/components/driver/twai/include -I/home/user/tools/esp-idf/components/driver/uart/include -I/home/user/tools/esp-idf/components/driver/usb_serial_jtag/include -I/home/user/tools/esp-idf/components/driver/touch_sensor/esp32/include -I/home/user/tools/esp-idf/components/esp_pm/include -I/home/user/tools/esp-idf/components/esp_ringbuf/include -I/home/user/matter-esp32-dcmotor/managed_components/espressif__button/include -I/home/user/tools/esp-idf/components/esp_adc/include -I/home/user/tools/esp-idf/components/esp_adc/interface -I/home/user/tools/esp-idf/components/esp_adc/esp32/include -I/home/user/tools/esp-idf/components/esp_adc/deprecated/include -I/home/user/tools/esp-matter/components/esp_matter -I/home/user/tools/esp-matter/components/esp_matter/utils -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/zzz_generated/app-common -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/third_party/nlfaultinjection/include -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/src -I/home/user/tools/esp-matter/components/esp_matter/zap_common -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/src/platform/ESP32 -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/src/platform/OpenThread -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/src/include -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/src/lib -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/examples/platform/esp32 -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/third_party/nlassert/repo/include -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/third_party/nlio/repo/include -I/home/user/matter-esp32-dcmotor/build/esp-idf/chip/src/include -I/home/user/matter-esp32-dcmotor/build/esp-idf/chip/include -I/home/user/matter-esp32-dcmotor/build/esp-idf/chip/gen/include -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/config/esp32 -I/home/user/tools/esp-matter/connectedhomeip/connectedhomeip/config/esp32/../.. -I/home/user/matter-esp32-dcmotor/managed_components/espressif__esp_encrypted_img/include -I/home/user/matter-esp32-dcmotor/managed_components/espressif__esp_insights/include -I/home/user/matter-esp32-dcmotor/managed_components/espressif__esp_diagnostics/include -I/home/user/tools/esp-idf/components/bt/include/esp32/include -I/home/user/tools/esp-idf/components/bt/common/osi/include -I/home/user/tools/esp-idf/components/bt/common/api/include/api -I/home/user/tools/esp-idf/components/bt/common/btc/profile/esp/blufi/include -I/home/user/tools/esp-idf/components/bt/common/btc/profile/esp/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/ans/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/bas/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/dis/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/gap/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/gatt/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/hr/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/htp/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/ias/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/ipss/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/lls/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/prox/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/services/tps/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/util/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/store/ram/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/host/store/config/include -I/home/user/tools/esp-idf/components/bt/porting/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/porting/nimble/include -I/home/user/tools/esp-idf/components/bt/host/nimble/port/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/transport/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/porting/npl/freertos/include -I/home/user/tools/esp-idf/components/bt/host/nimble/nimble/nimble/include -I/home/user/tools/esp-idf/components/bt/host/nimble/esp-hci/include -I/home/user/tools/esp-idf/components/esp_timer/include -I/home/user/tools/esp-idf/components/esp_wifi/include -I/home/user/tools/esp-idf/components/esp_wifi/wifi_apps/include -I/home/user/tools/esp-idf/components/esp_event/include -I/home/user/tools/esp-idf/components/esp_phy/include -I/home/user/tools/esp-idf/components/esp_phy/esp32/include -I/home/user/tools/esp-idf/components/esp_netif/include -I/home/user/tools/esp-matter/components/esp_matter_console -I/home/user/tools/esp-idf/components/nvs_flash/include -I/home/user/tools/esp-idf/components/spi_flash/include -I/home/user/tools/esp-idf/components/esp_partition/include -I/home/user/tools/esp-idf/components/app_update/include -I/home/user/tools/esp-idf/components/bootloader_support/include -I/home/user/tools/esp-idf/components/bootloader_support/bootloader_flash/include -I/home/user/tools/esp-idf/components/esp_app_format/include -I/home/user/matter-esp32-dcmotor/managed_components/espressif__esp_secure_cert_mgr/include -I/home/user/tools/esp-idf/components/mbedtls/port/include -I/home/user/tools/esp-idf/components/mbedtls/mbedtls/include -I/home/user/tools/esp-idf/components/mbedtls/mbedtls/library -I/home/user/tools/esp-idf/components/mbedtls/esp_crt_bundle/include -I/home/user/tools/esp-idf/components/efuse/include -I/home/user/tools/esp-idf/components/efuse/esp32/include -I/home/user/tools/esp-idf/components/json/cJSON -I/home/user/tools/esp-matter/components/esp_matter_bridge -I/home/user/tools/esp-matter/examples/common/app_reset -mlongcalls -Wno-frame-address  -ffunction-sections -fdata-sections -Wall -Werror=all -Wno-error=unused-function -Wno-error=unused-variable -Wno-error=unused-but-set-variable -Wno-error=deprecated-declarations -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-enum-conversion -gdwarf-4 -ggdb -Os -freorder-blocks -fmacro-prefix-map=/home/user/matter-esp32-dcmotor=. -fmacro-prefix-map=/home/user/tools/esp-idf=/IDF -fstrict-volatile-bitfields -fno-jump-tables -fno-tree-switch-conversion -DconfigENABLE_FREERTOS_DEBUG_OCDAWARE=1 -Wno-error=uninitialized -Wno-error=maybe-uninitialized -Wno-missing-field-initializers -Wno-error=array-bounds -Wno-write-strings -Wno-unused-function -Wno-format-nonliteral -Wno-format-security -Wformat=0 -std=gnu++2b -fno-exceptions -fno-rtti -std=gnu++17 -DCHIP_HAVE_CONFIG_H -fpermissive "-DCHIP_ADDRESS_RESOLVE_IMPL_INCLUDE_HEADER=<lib/address_resolve/AddressResolve_DefaultImpl.h>" -MD -MT esp-idf/main/CMakeFiles/__idf_main.dir/src/system/util.cpp.obj -MF esp-idf/main/CMakeFiles/__idf_main.dir/src/system/util.cpp.obj.d -o esp-idf/main/CMakeFiles/__idf_main.dir/src/system/util.cpp.obj -c /home/user/matter-esp32-dcmotor/main/src/system/util.cpp
/home/user/matter-esp32-dcmotor/main/src/system/util.cpp: In function 'std::string get_matter_attribute_value_string(esp_matter_attr_val_t)':
/home/user/matter-esp32-dcmotor/main/src/system/util.cpp:499:12: error: enumeration value 'ESP_MATTER_VAL_TYPE_NULLABLE_BOOLEAN' not handled in switch [-Werror=switch]
  499 |     switch (value.type) {
      |            ^
cc1plus: some warnings being treated as errors
ninja: build stopped: subcommand failed.
ninja failed with exit code 1, output of the command is in the /home/user/matter-esp32-dcmotor/build/log/idf_py_stderr_output_42456 and /home/user/matter-esp32-dcmotor/build/log/idf_py_stdout_output_42456
```

--- a/main/src/system/util.cpp
+++ b/main/src/system/util.cpp
@@ -501,6 +501,7 @@ std::string get_matter_attribute_value_string(esp_matter_attr_val_t value)
         snprintf(temp, sizeof(temp), "invalid");
         break;
     case ESP_MATTER_VAL_TYPE_BOOLEAN:
+    case ESP_MATTER_VAL_TYPE_NULLABLE_BOOLEAN:
         snprintf(temp, sizeof(temp), "%d", value.val.b);
         break;
     case ESP_MATTER_VAL_TYPE_INTEGER:

